src/components/projects_home/projectsCard.tsx

```diff
@@ -0,0 +1,178 @@
+import React from 'react';
+import style from './projectsCard.module.sass';
+import { useHistory, Link } from 'react-router-dom';
+
+function renderPriority(level: string) {
```

Comment:
This function takes in the projects priority and based on the level, it renders a small component.